jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -883,15 +883,14 @@ YUI.add('juju-topology-service', function(Y) {
      * @return {undefined} Nothing.
      */
     _deployLocalCharm: function(file, env, db) {
-      var topo = this.get('component');
-      topo.get('state').changeState({
+      // Store the local file in the global store.
+      window.localCharmFile = file;
```

Comment:
Nope it's not. I suppose I could add it to the DF to clear the file out if it exists.